pkg/api/api.go

```diff
@@ -229,11 +230,12 @@ func (a *API) RegisterQuerier(svc querierv1connect.QuerierServiceHandler) {
 	querierv1connect.RegisterQuerierServiceHandler(a.server.HTTP, svc, a.grpcAuthMiddleware, a.grpcLogMiddleware)
 }
 
-func (a *API) RegisterPyroscopeHandlers(client querierv1connect.QuerierServiceClient) {
-	handlers := querier.NewHTTPHandlers(client)
+func (a *API) RegisterPyroscopeHandlers(qc querierv1connect.QuerierServiceClient, sc settingsv1connect.SettingsServiceClient) {
+	handlers := querier.NewHTTPHandlers(qc, sc)
 	a.RegisterRoute("/pyroscope/render", http.HandlerFunc(handlers.Render), true, true, "GET")
 	a.RegisterRoute("/pyroscope/render-diff", http.HandlerFunc(handlers.RenderDiff), true, true, "GET")
 	a.RegisterRoute("/pyroscope/label-values", http.HandlerFunc(handlers.LabelValues), true, true, "GET")
+	a.RegisterRoute("/pyroscope/settings", http.HandlerFunc(handlers.Settings), true, true, "GET", "POST")
```

Comment:
Can't we use connect-go API only ? those API `render` and such are here for backward compatbility reason.

Reply:
Yeah good call. I did that. This was the first pass of me hacking around and using the connect-go API only is the way to go.